scripts/battlefields/Throne_Room/kindred_spirits.lua

```diff
@@ -0,0 +1,84 @@
+-----------------------------------
+-- Area: Throne Room
+-- BCNM 60 - Kindred Spirits
+-- !pos -111 -6 0.1 165
+-----------------------------------
+local ID = require("scripts/zones/Throne_Room/IDs")
+require("scripts/globals/battlefield")
+require("scripts/globals/items")
+require("scripts/globals/zone")
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.THRONE_ROOM,
+    battlefieldId = xi.battlefield.id.KINDRED_SPIRITS,
+    maxPlayers    = 6,
+    levelCap      = 60,
+    timeLimit     = utils.minutes(30),
+    index         = 2,
+    entryNpc      = "_4l1",
+    exitNpcs      = { "_4l2", "_4l3", "_4l4" },
+    requiredItems = { xi.items.MOON_ORB, message = ID.text.A_CRACK_HAS_FORMED },
```

Comment:
This should fix it so that the orb wears upon use and will display a worn message when you try to use a worn orb.
```suggestion
    requiredItems = { xi.items.MOON_ORB, wearMessage = ID.text.A_CRACK_HAS_FORMED, wornMessage = ID.text.ORB_CRACKED },
```